scripts/Gossip.js

```diff
@@ -40,28 +40,42 @@ var Gossip = function(robot) {
     });
   }
 
-  var findGossip = function(text, msg) {
+  var findGossip = function(text, msg, limiter) {
     var connection = connectionHelper.getConnection();
     var query = 'SELECT * FROM gossip';
     var queryArgs = [];
+    var gossip = '\n';
 
     if (text != null && text.length > 0) {
       query += ' WHERE gossip_text LIKE ?';
       queryArgs.push('%' + text + '%');
     }
 
-    query += ' ORDER BY RAND() LIMIT 0,1';
+    query += ' ORDER BY RAND() LIMIT ' + (limiter || 1);
 
     connection.query(query, queryArgs, function(err, rows) {
       if (err) {
         msg.reply('Something broke');
       }
 
-      if (rows.length === 0) {
+      if (rows.length === 0 && !limiter) {
```

Comment:
`limiter` is an integer, not a bool.  Try to avoid things like this, and prefer `limiter > 0`